examples/braket_features/Noise_models/Noise_models_on_Rigetti_AspenM1.ipynb

```diff
@@ -0,0 +1,1057 @@
+{
```

Comment:
<p>"Braket console provides calibration data for each QPU" should we change to "Braket and the Braket Console vends varying levels of calibration data for each QPU"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
hmm, I'm not sure how to best word this. Could it be a bit more specific like: "Braket provides access to hardware providers reported calibration data"?

Reply:
Yes, I like that phrasing!